### ISSUE TITLE
Warning: Scrolling(div): Support for defaultProps will be removed from function components in a future major release

### ISSUE THREAD

**Issue by vasiline2256** [TechStark/react-dnd-scrolling#32]
```
Warning: Scrolling(div): Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.
    at ScrollingComponent (webpack-internal:///./node_modules/react-dnd-scrolling/lib/esm/index.js:96:36)
    at DndProvider (webpack-internal:///./node_modules/react-dnd/dist/core/DndProvider.js:43:11)
    at MyApp (webpack-internal:///./pages/_app.js:148:11)
    at PathnameContextProviderAdapter (webpack-internal:///./node_modules/next/dist/shared/lib/router/adapters.js:62:11)
    at ErrorBoundary (webpack-internal:///./node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:301:63)
    at ReactDevOverlay (webpack-internal:///./node_modules/next/dist/compiled/@next/react-dev-overlay/dist/client.js:850:919)
    at Container (webpack-internal:///./node_modules/next/dist/client/index.js:61:1)
    at AppContainer (webpack-internal:///./node_modules/next/dist/client/index.js:171:11)
    at Root (webpack-internal:///./node_modules/next/dist/client/index.js:346:11)
```

Hi, anyone know how to fix this?

**Comment by ttt43ttt** [comment]
what's your source code around usage of `<DndProvider>`?

**Comment by ankursehdev** [comment]
I am using following versions and can see this error on console. 

```
 "react-dnd": "^16.0.1",
    "react-dnd-html5-backend": "^16.0.1",
    "react-dnd-scrolling": "^1.3.8",
```

**Comment by ttt43ttt** [comment]
This should be fixed after the next release https://github.com/TechStark/react-dnd-scrolling/commit/883d63871eada051ae30a49ecba8649a8f634c86#diff-ec767e14d91498b5b31a4ce826c4596061eb83a47d3dee00f6cbe1078232a75cL113

**Comment by kurkle** [comment]
> This should be fixed after the next release [883d638#diff-ec767e14d91498b5b31a4ce826c4596061eb83a47d3dee00f6cbe1078232a75cL113](https://github.com/TechStark/react-dnd-scrolling/commit/883d63871eada051ae30a49ecba8649a8f634c86#diff-ec767e14d91498b5b31a4ce826c4596061eb83a47d3dee00f6cbe1078232a75cL113)

any plans on releasing?

**Comment by ttt43ttt** [comment]
please try `react-dnd-scrolling@1.3.9`
